airflow/providers/microsoft/azure/hooks/data_lake.py

```diff
@@ -308,18 +310,18 @@ def get_conn(self) -> DataLakeServiceClient:  # type: ignore[override]
             # connection_string auth takes priority
             return DataLakeServiceClient.from_connection_string(connection_string, **extra)
 
+        credential: Credentials
         tenant = self._get_field(extra, "tenant_id")
         if tenant:
             # use Active Directory auth
             app_id = conn.login
             app_secret = conn.password
-            token_credential = ClientSecretCredential(tenant, app_id, app_secret)
-            return DataLakeServiceClient(
-                account_url=f"https://{conn.host}.dfs.core.windows.net", credential=token_credential, **extra
-            )
+            credential = ClientSecretCredential(tenant, app_id, app_secret)
+        elif conn.password:
+            credential = conn.password
+        else:
+            credential = DefaultAzureCredential()
```

Comment:
Can you post a screenshot of a task working successfully with the `DefaultAzureCredential` that you introduced

Reply:
Sure. I'll test and post a screenshot before I mark these PRs as read to review